browser/src/control/Control.UIManager.ts

```diff
@@ -908,6 +908,8 @@ class UIManager extends L.Control {
 			this.insertButtonToClassicToolbar(button);
 		else
 			this.notebookbar.insertButtonToShortcuts(button);
+		if (button.target === 'context-toolbar')
```

Comment:
ok, but it will add also to the notebookbar or toolbar ^

we should check if target is set first possibly, if not -> add to the main UI

Reply:
ah, i assumed that we also need that in the toolbar.